e2e/specs/startup/app.test.js

```diff
@@ -41,10 +47,10 @@ describe('startup/app', function desc() {
         existingModal.should.not.be.null;
     });
 
-    it('MM-T4399_2 should show no servers configured in dropdown when no servers exist', async () => {
```

Comment:
@devinbinnie Do you know if this step 2 previously exist? I've checked [here](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T4399) but only one step was there.